pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -116,9 +118,27 @@ func (a *ApplicationRestoreController) setDefaults(restore *storkapi.Application
 			restore.Spec.NamespaceMapping[ns] = ns
 		}
 	}
+
 	return nil
 }
 
+func getRancherProjectMapping(restore *storkapi.ApplicationRestore) map[string]string {
+	rancherProjectMapping := map[string]string{}
+	if restore.Spec.RancherProjectMapping != nil {
+		for key, value := range restore.Spec.RancherProjectMapping {
+			rancherProjectMapping[key] = value
+			dataKey := strings.Split(key, ":")
```

Comment:
As i understand during the backup itself, we have taken relevant projects and stored in object. under which condition would this have value with `:` as delimeter?

Reply:
No, during the backup we store in the format clusterID:projectID, please refer the previous comment.